### PR TITLE
CL Week Two Assign Code review updates

### PR DESCRIPTION
**Pull request by Devansu-Yadav**
This is a PR to Re-factor code based on the Week Two assignment Review/suggested changes to the `temp-dev` branch.

- Fixed the CTA button on the Home page
- Removed console logs and unnecessary comments in code.
- Fixed the close button in modal.
- Removed `<br>` tags from all pages and made the spacing more consistent on all pages of the Documentation.
- Added updated Demo video for the newly added P2 components (Navigation & Slider)

Preview link for the changes - https://deploy-preview-20--sapphire-ui.netlify.app/

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
Everything looks good to me
good to merge

**Review by mdarif6 — APPROVED** [review]
- Complete CL is looking nice.
- Good to see few working components with JavaScript.
- Simple Explanation.

File: Components/Modal/modal.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-        document.body.style.overflow = "hidden";
-        modalOverlayBg.style.display = "block";
-        modalOverlayBg.style.backgroundColor = "var(--modal-overlay-color)";
+        showModal();
     } else {
-        modal.classList.remove("show");
-        document.body.style.overflow = "auto";
-        modalOverlayBg.style.display = "none";
-        modalOverlayBg.style.backgroundColor = "var(--white-color)";
+       hideModal();
```
Comment (tejas-muthya37) [inline Components/Modal/modal.js:25]: Try if you can write a single function called toggleModal.

File: Docs/nav-menu.js [hunk 2]
```diff
@@ -5,12 +5,10 @@ hamburgerMenuBtn.addEventListener("click", function() {
     if(hamburgerMenuBtn.classList.contains("fa-bars")) {
         hamburgerMenuBtn.classList.remove("fa-bars");
```
Comment (tejas-muthya37) [inline Docs/nav-menu.js:6]: Feel like you can use classList.toggle here.

**Review by tejas-muthya37 — APPROVED** [review]
Since most of the PR is HTML markup, there are not a lot of issues that I could spot.

3 things that I liked,

Consistency in naming conventions.
Use of JavaScript for interactive behavior.
Short and detailed code snippets.

My suggestion would be to keep PRs short and detailed.

**Pull request merged**